index.php

```diff
@@ -62,11 +62,6 @@
 }
 error_reporting($level);
 
-if (PHP_VERSION_ID < 50407)
-{
-	gen_error_msg('ERROR_PHP_UNSUPPORTED');
-}
-
 // If we are on PHP5 we may need to define STRIP to strip slashes
 define('STRIP', PHP_VERSION_ID < 60000 && get_magic_quotes_gpc());
```

Comment:
PHP version 6 has been never released though.

Reply:
Yes but this is how you make sure you're checking for PHP5 or less